### PR TITLE
Var rename 02

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: src/main/java/games/strategy/engine/framework/ServerGame.java [hunk 1]
```diff
@@ -238,7 +238,7 @@ public static RemoteName getRemoteRandomName(final PlayerID id) {
 
   private GameStep getCurrentStep() {
     return m_data.getSequence().getStep();
-    // m_data.getSequence().getStep(m_currentStepIndex);
+    // gameData.getSequence().getStep(m_currentStepIndex);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/engine/framework/ServerGame.java]: This comment should be removed

File: src/main/java/games/strategy/triplea/delegate/PoliticsDelegate.java [hunk 2]
```diff
@@ -379,7 +379,7 @@ private void changeRelationships(final PoliticalActionAttachment paa) {
        * creation of new battles is handled at the beginning of the battle delegate, in
        * "setupUnitsInSameTerritoryBattles", not here.
        * if (Matches.RelationshipTypeIsAtWar.match(newRelation))
-       * TriggerAttachment.triggerMustFightBattle(player1, player2, m_bridge);
+       * TriggerAttachment.triggerMustFightBattle(player1, player2, playerBridge);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/delegate/PoliticsDelegate.java]: This comment should be removed (at least the code part of it)

File: src/main/java/games/strategy/triplea/delegate/TechActivationDelegate.java [hunk 3]
```diff
@@ -50,7 +50,7 @@ public void start() {
       // Start event
       m_bridge.getHistoryWriter().startEvent(m_player.getName() + " activating " + advancesAsString(advances));
       for (final TechAdvance advance : advances) {
-        // advance.perform(m_bridge.getPlayerID(), m_bridge, m_data);
+        // advance.perform(playerBridge.getPlayerID(), playerBridge, gameData);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/delegate/TechActivationDelegate.java]: This comment should be removed

File: src/main/java/games/strategy/triplea/delegate/TechnologyDelegate.java [hunk 4]
```diff
@@ -416,7 +416,7 @@ private int getTechHits(final int[] random) {
   }
 
   private List<TechAdvance> getAvailableAdvancesForCategory(final TechnologyFrontier techCategory) {
-    // Collection<TechAdvance> allAdvances = TechAdvance.getTechAdvances(m_data, techCategory);
+    // Collection<TechAdvance> allAdvances = TechAdvance.getTechAdvances(gameData, techCategory);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/delegate/TechnologyDelegate.java]: This comment should be removed

File: src/main/java/games/strategy/triplea/ui/AbstractMovePanel.java [hunk 5]
```diff
@@ -120,7 +120,7 @@ protected final void enableCancelButton() {
   }
 
   /**
-   * @return m_bridge.getGameData()
+   * @return playerBridge.getGameData()
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/ui/AbstractMovePanel.java]: Wrong javadoc change

File: src/main/java/games/strategy/triplea/ui/TripleAFrame.java [hunk 6]
```diff
@@ -1728,7 +1728,7 @@ public void actionPerformed(final ActionEvent ae) {
                 + "such as a move or battle phase.",
             "Save Game from History", JOptionPane.INFORMATION_MESSAGE);
         data.acquireReadLock();
-        // m_data.acquireWriteLock();
+        // gameData.acquireWriteLock();
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/ui/TripleAFrame.java]: This comment should be removed

File: src/main/java/games/strategy/triplea/ui/StatPanel.java [hunk 7]
```diff
@@ -388,7 +388,7 @@ public void update() {
               data[row][col] = "-";
             }
           }
-          final Iterator<TechAdvance> advances = TechTracker.getCurrentTechAdvances(pid, m_data).iterator();
+          final Iterator<TechAdvance> advances = TechTracker.getCurrentTechAdvances(pid, StatPanel.this.gameData).iterator();
```
Comment (ssoloff) [inline src/main/java/games/strategy/triplea/ui/StatPanel.java]: Checkstyle: "Line is longer than 120 characters (found 125)."

File: src/main/java/games/strategy/triplea/ui/StatPanel.java [hunk 8]
```diff
@@ -141,15 +141,15 @@ protected ImageIcon getIcon(final PlayerID player) {
   }
 
   protected ImageIcon getIcon(final String playerName) {
-    final PlayerID player = this.m_data.getPlayerList().getPlayerID(playerName);
+    final PlayerID player = this.gameData.getPlayerList().getPlayerID(playerName);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/ui/StatPanel.java]: explicit this is not needed

File: src/main/java/games/strategy/triplea/ui/StatPanel.java [hunk 9]
```diff
@@ -352,7 +352,7 @@ private void clearAdvances() {
     }
 
     private void initColList() {
-      final java.util.List<PlayerID> players = new ArrayList<>(m_data.getPlayerList().getPlayers());
+      final java.util.List<PlayerID> players = new ArrayList<>(gameData.getPlayerList().getPlayers());
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/ui/StatPanel.java]: List should probably be imported

**Comment by DanVanAtta** [comment]
Thanks for quick review @RoiEXLab , now updated, merging pending tests (updates were pretty trivial)

File: src/main/java/games/strategy/triplea/ui/StatPanel.java (hunk 9 above)
Comment (DanVanAtta) [inline src/main/java/games/strategy/triplea/ui/StatPanel.java]: Even better, List was already imported, quick fix included: e346031

**Pull request merged**